taskograph.EF/Repositories/ColorRepository.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Microsoft.Extensions.Logging;
+using Task = taskograph.Models.Tables.Task;
+using Group = taskograph.Models.Tables.Group;
+using Color = taskograph.Models.Tables.Color;
+using static taskograph.Helpers.Messages;
+using taskograph.EF.DataAccess;
+using taskograph.EF.Repositories.Infrastructure;
+using taskograph.Models.Tables;
+using System.Drawing;
+using Microsoft.EntityFrameworkCore;
+
+namespace taskograph.EF.Repositories
+{
+    public class ColorRepository : IColorRepository
+    {
+        private readonly TasksContext _db;
+        private readonly ILogger<ColorRepository> _logger;
+
+        public ColorRepository(TasksContext db, ILogger<ColorRepository> logger)
+        {
+            _db = db;
+            _logger = logger;
+        }
+
+
+        public bool Add(string name)
```

Comment:
A może zamiast tworzyć nowy model z parametrem nazwy to spróbuj zbindować parametr do modelu? Do tego widzę, że tworzysz repozytorium trochę na sucho, jeśli nic nie pominęłam, to ono nie ma podpięcia do niczego? Pamiętaj o YAGNI ;)